ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -53,15 +53,15 @@ const (
 	requestTimedOutError       = "%s: request timed out"
 	// This is our initial assumption of how much time it would take for the Linux commands used to inject faults
 	// to finish. This will be confirmed/updated after more testing.
-	requestTimeoutDuration = 5 * time.Second
+	requestTimeoutDuration = 5
```

Comment:
nit: now that we have removed unit, maybe we should rename this to `requestTimeoutSeconds`

Reply:
yep, good idea